### DISCUSSION TITLE
File uploads with Laravel Livewire

### DISCUSSION THREAD

**Discussion by abishekrsrikaanth** [spatie/laravel-medialibrary#1933] [Q&A] [answered]
@freekmurze, I have been trying to get this package working with Laravel Livewire and so far I have not succeeded. Would you be able to share any examples of how this needs to be done? With the latest release of Laravel Livewire with support for `FileUploads`, it always throws an error that the File Was not found even after defining the right paths on the config.

Appreciate if you can help in regards to this.

**Comment by freekmurze** [comment, marked as answer]
I'm currently working on creating first class support for Livewire. It will be part of the upcoming https://medialibrary.pro package, which will be released in the next couple of months.

**Reply by abishekrsrikaanth** [reply]
Does that mean, the current state of the package is not usable with livewire? I have 2 projects that are currently working on migration to livewire but unfortunately, at a show stopper with file uploads using this package. Any help is appreciated.

**Reply by freekmurze** [reply]
It is usable with Livewire. The current media library doesn't care about how you handle uploads. Medialibrary pro will offer very easy to use Blade components that wrap up Livewire functionality.

**Reply by abishekrsrikaanth** [reply]
Alright, thank you.

**Comment by marbuser** [comment]
@freekmurze Any update on this at all? I'm also in need of this for a project and it's already been 2 months without many updates on this. :/

**Reply by freekmurze** [reply]
I'm still actively working on this, but need more time to polish it. ETA now would be end September / October.
